### DISCUSSION TITLE
Cable Bulk Import - circuits.circuittermination

### DISCUSSION THREAD

**Discussion by hjimenez-pm** [netbox-community/netbox#6840] [Q&A]
I would like to ask if we can upload circuits.circuittermination type in cables.
When uploading it shows error:

Enter the list of column headers followed by one line per record to be imported, using commas to separate values. Multi-line data and values containing commas may be wrapped in double quotes.
Row 1 side_b_device: Object not found.
Row 1 __all__: Termination B type has not been specified

But I don't have idea what could be the side_b_device since it's circuit termination.
I can upload dcim.interface for both side for devices without any issue.

NetBox: pttnetboxdev (v2.11.9)
In the screenshot below: Cable 31 and 33: Side A is the device. Side B is the Circuit ID.
![image](https://user-images.githubusercontent.com/84356047/127475148-9f49a164-01b1-4854-8a48-83227846e41c.png)

**Comment by ryanmerolle** [comment]
I recommend you model whatever you want to bulk import in netbox manually, and export the data so you can see what you would need to do to build for the bulk import.

**Reply by candlerb** [reply]
That doesn't really work, because the CSV export is sadly different to CSV import in many aspects.

**Reply by ryanmerolle** [reply]
It's not an exact header match but it should give a good example that you can edit when you import to match the import help/guidance.

That's at least how I have done it.

**Reply by candlerb** [reply]
Not just the column names, even the contents of some of the fields are different - e.g. the names to refer to the models.

**Comment by ryanmerolle** [comment]
True but I’ve still used it with the import tool tips to figure out what I needed for the bulk import.
